npm error with phantomjs version

Travis build on https://github.com/mozilla/teach.webmaker.org/pull/1106 failed due the following error

```bash
npm ERR! peerinvalid The package phantomjs does not satisfy its siblings' peerDependencies requirements!

npm ERR! peerinvalid Peer mocha-phantomjs@3.5.4 wants phantomjs@1.9.1 - 1.9.7-15
```
Reason:
[phantomjs](https://www.npmjs.com/package/phantomjs) is now at 1.9.17 which meets [our specification in package.json](https://github.com/mozilla/teach.webmaker.org/blob/v0.0.15/package.json#L55) but phantomjs@1.9.17 doesn't fall within [the range of version mocha-phantomjs takes](https://github.com/nathanboktae/mocha-phantomjs/blob/v3.6.0/package.json#L39).

Fix:
We should probably just match our semvar string fo phantomjs with mocha-phantomjs's for now.  I've created [this ticket](https://github.com/mozilla/teach.webmaker.org/issues/1115) to remind us to update the string once mocha-phantomjs update theirs.

/cc @simonwex

--- a/package.json
+++ b/package.json
@@ -52,7 +52,7 @@
   },
   "devDependencies": {
     "mocha-phantomjs": "^3.5.3",
-    "phantomjs": "^1.9.16",
+    "phantomjs": "1.9.1 - 1.9.7-15",
     "simplecrawler": "^0.5.2",
     "supertest": "^1.0.1",
     "marked": "^0.3.3",

